biz.aQute.bndlib/src/aQute/bnd/differ/DiffPluginImpl.java

```diff
@@ -100,9 +100,8 @@ public Tree tree(File newer) throws Exception {
 	 * @see aQute.bnd.service.diff.Differ#tree(aQute.bnd.osgi.Jar)
 	 */
 	public Tree tree(Jar newer) throws Exception {
-		Analyzer anewer = new Analyzer();
+		Analyzer anewer = new Analyzer(newer);
 		try {
-			anewer.setJar(newer);
 			return tree(anewer);
 		} finally {
 			anewer.setJar((Jar) null);
```

Comment:
You can remove the `setJar(null)`.

Might also want to tidy this up into a try-with-resources block to avoid the finally with a close call.

```java
try (Analyzer anewer = new Analyzer(newer)) {
  return tree(anewer);
}
```